src/api/routes/v1/validator.ts

```diff
@@ -185,6 +188,7 @@ async function formatResponse(resp: dbResponse): Promise<ValidatorResponse> {
     server_version: resp.server_version,
     networks: resp.networks,
     current_index: Number(resp.current_index),
+    ledger_hash: resp.ledger_hash,
```

Comment:
`current_hash` or `current_ledger_hash` would be more informative imo. We can just fix on API level here

Reply:
I don't mind changing it, however users will not find any documentation on XRPL.org for `current_hash` (or) `current_ledger_hash`.

If you feel strongly, I can change it.

Reply:
Furthermore, the [`validators` table also has a column](https://github.com/ripple/validator-history-service/blob/main/ARCHITECTURE.md#validators) with the name `ledger_hash`. It would be better to use the same name across all the tables, otherwise it is confusing.

Reply:
I'm not suggesting we change name of table, but on the API result there's a `current_index` so I just want to call `ledger_hash` as `current_hash` on the API front so that it makes more sense when users see it. Will leave the call to you then since this is minor

Reply:
I see. I was not aware of the current_index API result, I agree that this will be an inconsistency. 👍 
However, I believe it is better to be consistent with the documentation on xrpl.org, instead of the VHS API. I'd prefer to retain this name, if that's alright.

Thanks for explaining the difference 👍